src/components/spreadsheet/export-csv.js

```diff
@@ -40,10 +40,11 @@ export const CsvExport = ({
                     ?.getModel()
                     ?.rowsToDisplay.map((node) => node.data) || [];
             Object.keys(gridData).forEach((item) => {
-                gridData[item].limitName = formatNAValue(
-                    gridData[item].limitName,
-                    intl
-                );
+                gridData[item].limitName &&
+                    (gridData[item].limitName = formatNAValue(
+                        gridData[item].limitName,
+                        intl
+                    ));
```

Comment:
I had previous PR with review asking to use more "classic" syntax before...
```suggestion
                if (gridData[item].limitName) {
                    gridData[item].limitName = formatNAValue(
                        gridData[item].limitName,
                        intl
                    );
                }
```
but it's ok for me